tests/system/providers/google/cloud/dataplex/example_dataplex.py

```diff
@@ -21,25 +21,41 @@
 
 import datetime
 import os
+from pathlib import Path
 
 from airflow import models
 from airflow.models.baseoperator import chain
 from airflow.providers.google.cloud.operators.dataplex import (
+    DataplexCreateLakeOperator,
     DataplexCreateTaskOperator,
+    DataplexDeleteLakeOperator,
     DataplexDeleteTaskOperator,
     DataplexGetTaskOperator,
     DataplexListTasksOperator,
 )
+from airflow.providers.google.cloud.operators.gcs import GCSCreateBucketOperator, GCSDeleteBucketOperator
 from airflow.providers.google.cloud.sensors.dataplex import DataplexTaskStateSensor
+from airflow.providers.google.cloud.transfers.local_to_gcs import LocalFilesystemToGCSOperator
+from airflow.utils.trigger_rule import TriggerRule
 
-PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "INVALID PROJECT ID")
-REGION = os.environ.get("GCP_REGION", "INVALID REGION")
-LAKE_ID = os.environ.get("GCP_LAKE_ID", "INVALID LAKE ID")
-SERVICE_ACC = os.environ.get("GCP_DATAPLEX_SERVICE_ACC", "XYZ@developer.gserviceaccount.com")
-BUCKET = os.environ.get("GCP_DATAPLEX_BUCKET", "INVALID BUCKET NAME")
-SPARK_FILE_NAME = os.environ.get("SPARK_FILE_NAME", "INVALID FILE NAME")
-SPARK_FILE_FULL_PATH = f"gs://{BUCKET}/{SPARK_FILE_NAME}"
-DATAPLEX_TASK_ID = "task001"
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT")
+
+DAG_ID = "example_dataplex"
+
+BUCKET_NAME = f"bucket_{DAG_ID}_{ENV_ID}"
+
+SPARK_FILE_NAME = "spark_example_pi.py"
+CURRENT_FOLDER = Path(__file__).parent
+FILE_LOCAL_PATH = str(Path(CURRENT_FOLDER) / "resources" / SPARK_FILE_NAME)
+
+LAKE_ID = "lake-test"
+REGION = "us-central1"
+
+SERVICE_ACC = os.environ.get("GCP_DATAPLEX_SERVICE_ACC")
+
+SPARK_FILE_FULL_PATH = f"gs://{BUCKET_NAME}/{SPARK_FILE_NAME}"
+DATAPLEX_TASK_ID = "test-task"
```

Comment:
Maybe it should use ENV_ID in the name? so it would be bit more unique name

Reply:
changed